library/pom.xml

```diff
@@ -294,6 +294,13 @@
 			<artifactId>viewerjs</artifactId>
 			<version>0.5.8</version>
 		</dependency>
+		<dependency>
```

Comment:
Would you mind documenting in what tools this webjar is used? There are some examples in this file.

Thanks again! I've approved it.